pkg/operator/status.go

```diff
@@ -607,6 +607,14 @@ func isMachineConfigPoolConfigurationValid(pool *mcfgv1.MachineConfigPool, versi
 	if renderedMC.Spec.OSImageURL != osURL {
 		return fmt.Errorf("osImageURL mismatch for %s in %s expected: %s got: %s", pool.GetName(), renderedMC.Name, osURL, renderedMC.Spec.OSImageURL)
 	}
+
+	// check that the rendered config matches the OCP release version for cases where there is no OSImageURL change nor no new MCO commit
+	if !inClusterBringup {
```

Comment:
Why did you add this conditional?  The `inClusterBringup` bit came from https://github.com/openshift/machine-config-operator/commit/b8876d946f8bb87e95ce59efb7a88c3ff19fa43e and ...I no longer recall exactly what that variable means now :cry:   We should add a documentation comment for it.

I'm not saying it's not necessary, I just want to understand why it might be.

Reply:
Double checking on this, in previous iterations I hit a race when it would not be able to pass this check when the cluster was coming up. But now I'm wondering if it was more bc I was passing in the wrong value for releaseVersion. Let me try and will report back.

Reply:
Test runs showed successful install without this check so removed.